### DISCUSSION TITLE
Animated gifs

### DISCUSSION THREAD

**Discussion by thedinz** [photoview/photoview#916] [Q&A and Troubleshooting]
Sorry if this has come up before but are there either any plans to support animated gifs or is it supported already and perhaps my settings are to blame for them not showing?

**Comment by kkovaletp** [comment]
I've just tested the scan job in a folder with several gifs and got "File is not a supported media /photos/jorrbox-jorrparivar.gif" in the log for each. So, it looks like gifs are not currently supported. Feel free to create a PR to add the support if you have enough programming skills)
